catalog/forms.py

```diff
@@ -1,7 +1,65 @@
 from django import forms
+from django.core.exceptions import ValidationError
 
+from catalog.custom_validators import Validator
+from catalog.models import Product
 
 class ContactForm(forms.Form):
     user_name = forms.CharField(label="Имя", max_length=100)
     user_email = forms.EmailField(label="Электронная почта")
     user_text = forms.CharField(label="Сообщение", widget=forms.Textarea)
+
+
+class ProductForm(forms.ModelForm):
+    MAX_UPLOAD_SIZE = 5242880 #5MB
```

Comment:
Нагляднее написать так
```suggestion
    MAX_UPLOAD_SIZE = 5 * 1024 ** 2
```
Тогда и коммент не нужен (2 степень укажет на мегабайты)